### ISSUE TITLE
Read hcl optional attribute

### ISSUE THREAD

**Issue by jmcshane** [rodaine/hclencoder#23]
I know that this library is reading its own `omitempty` tag, but the hcl tag now supports an optional attribute. Could this library be updated to read that tag as well?

**Comment by rodaine** [comment]
Can you point me to this optionality? I can't seem to find it in the HCL docs. Is this a HCLv2 only feature?

**Comment by jmcshane** [comment]
yeah, ive realized that this library is specifically focused on hclv1. I'm looking at possibly writing out a version of this for v2 using this structure as a model. you can feel free to close this issue if you don't want to support this attribute
